content/en/servers/blocks/crud/index.md

```diff
@@ -0,0 +1,31 @@
++++
+title = 'CRUD'
+headless = true
+time = 10
+facilitation = false
+emoji= '🧩'
+[objectives]
+    1='Define CRUD'
++++
+
+We will build a CRUD API. CRUD stands for **C**reate, **R**etrieve, **U**pdate, **D**elete. If you think about it, this is what most applications do:
+
+✨ **C**reate some "resources"
+🐕 **R**etrieve them (GET them)
+📨 **U**pdate them
+🗑️ **D**elete them
+
+### 🎯 Goal
+
+Our API will manage movie data. It will:
+
+✨ **C**reate a new movie,
+🐕 **R**etrieve a list of movies or a single movie
```

Comment:
```suggestion
For now, we'll focus on the first two of these. We'll make a server that provides an API for managing movie data. It will:

✨ **C**reate a new movie,
🐕 **R**etrieve a list of movies or a single movie
```